fuzz/json_array_fuzzer.dict

```diff
@@ -0,0 +1,19 @@
+"{"
+"}"
+","
+"["
+"]"
+","
+":"
+"e"
+"e+"
+"e-"
+"E"
+"E+"
+"E-"
+"\""
+"null"
+"1"
+"1.234"
+"3e4"
+"NaN"
```

Comment:
Would it be useful to include `Infinity`/`-Infinity` in the dictionaries?

Reply:
Yes, good point. Added.